app/js/app/app.js

```diff
@@ -721,7 +729,7 @@ define([
             api.notifications.respond({id: notification.id, response: response}, {});
 
             // if they respond with dismissed then it means we should show them the external link if there is one
```

Comment:
This comment probably needs updating, becasue dismiss really was never a good word choice, and now we've changed to `ACKNOWLEDGED` it's plain confusing!